v1.0/coordinateSpace.schema.json

```diff
@@ -0,0 +1,67 @@
+{
+  "_type": "https://openminds.ebrains.eu/sands/CoordinateSpace",
+  "description": "Structured information describing a coordinate space.",
+  "required": [
+    "alias",
+    "anatomicalOrientation",
+    "fullName",
+    "origin",
+    "releaseDate",
+    "unit",
+    "versionIdentifier"
+  ],
+  "properties": {
+    "alias": {
+      "type": "string",
+      "_instruction": "Enter the user-defined identifier for this coordinate space (e.g. a short name)."
+    },
+    "anatomicalOrientation": {
+      "_instruction": "Add the axes orientation denoted in standard anatomical terms of direction (in 2D or 3D).",
+      "_linkedTypes": [
+        "https://openminds.ebrains.eu/controlledTerms/Axis"
+      ]
+    },
+    "defaultImages": {
+      "type": "array",
+      "_instruction": "Add one or several images used as visual representation in this coordinate space.",
```

Comment:
"Add one or several images used as visual representation of this coordinate space." ??